templates/pubspecs/tools/serverpod_cli/pubspec.yaml

```diff
@@ -42,7 +42,9 @@ dependencies:
   pub_semver: ^2.1.4
 
 dev_dependencies:
+  build_runner: 2.3.3
```

Comment:
Let's discuss it when you are back.

Right now it is only used to create a mock class for tests. The generated file is also checked in so there shouldn't be a need to re-run the file generation for the test to work. We could also implement the mock class ourselves to remove this dependency.

The strict version is needed because we support Dart SDK 2.19 and build_runner 2.3.3 was the last version that supported Dart SDK 2.14 and above. The newer versions only support Dart SDK 3.0. Source: https://pub.dev/packages/build_runner/versions

Reply:
Implemented the MockClient without build runner and mockito now so I could remove those two dependencies. Take a look and se what you think.

I also reverted the touch on build runner in the `serverpod_test_shared` project as well since I don't need to interact with build runner in this PR anymore. But I think we might have a problem there supporting Dart SDK 2.19.